.travis.yml

```diff
@@ -53,8 +53,17 @@ jobs:
     - os: windows
 
   include:
+    - stage: Lint
+      name: Lint, via Black
+      python: 3.8
+      install:
+        - pip install black
```

Comment:
This surprises me but I haven't much with `poetry`. Does this mean that `black` is installed for everyone who install `jrnl`?

Reply:
(If so, I don't think `black` should be installed for "normal" users)

Reply:
If you look at `pyproject.toml`, you'll see that there are two sets of dependencies, the regular dependencies, and the dev dependencies. In our build pipeline, we install the dev dependencies since we need behave to run the tests (behave is also not installed for regular users).